cirq-core/cirq/protocols/mixture_protocol_test.py

```diff
@@ -74,12 +77,18 @@ def _has_unitary_(self):
         return NotImplemented
 
 
+class ReturnsMixtureOfReturnsUnitary:
+    def _mixture_(self):
+        return ((0.4, ReturnsUnitary()), (0.6, ReturnsUnitary()))
+
+
 @pytest.mark.parametrize(
     'val,mixture',
     (
-        (ReturnsValidTuple(), ((0.4, 'a'), (0.6, 'b'))),
-        (ReturnsNonnormalizedTuple(), ((0.4, 'a'), (0.4, 'b'))),
+        (ReturnsValidTuple(), ((0.4, a), (0.6, b))),
+        (ReturnsNonnormalizedTuple(), ((0.4, a), (0.4, b))),
         (ReturnsUnitary(), ((1.0, np.ones((2, 2))),)),
+        (ReturnsMixtureOfReturnsUnitary(), ((0.4, np.ones((2, 2))), (0.6, np.ones((2, 2))))),
```

Comment:
Nitty nit - not sure if `ReturnsUnitary` was intentionally made to give non-unitary matrix.  Please consider replacing with a valid unitary for sake of readability.

https://github.com/quantumlib/Cirq/blob/2037706377b55046ee1974e318ecbb4ca94d96b4/cirq-core/cirq/protocols/mixture_protocol_test.py#L64-L66.